app/views/admin/governments/index.html.erb

```diff
@@ -1,44 +1,47 @@
-<% page_title "Governments" %>
-<h1>Governments</h1>
+<% content_for :page_title, "Governments" %>
+<% content_for :title, "Governments" %>
 
 <% if can?(:manage, Government) %>
-  <p>
-    <%= link_to "Create a government", new_admin_government_path %>
-  </p>
+  <%= render "govuk_publishing_components/components/button", {
+    text: "Create a government",
+    href: new_admin_government_path,
+    secondary_solid: true,
+    margin_bottom: 6
+  } %>
 <% end %>
 
-<table class="governments table table-striped table-bordered add-top-margin">
-  <thead>
-    <tr class="table-header">
-      <th width="70%">Name</th>
-      <th width="15%">Start date</th>
-      <th width="15%">End date</th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <% @governments.each do |government| %>
-      <%= content_tag_for(:tr, government) do %>
-        <td>
-          <% if can?(:manage, Government) %>
-            <%= link_to government.name, edit_admin_government_path(government) %>
-          <% else %>
-            <%= government.name %>
-          <% end %>
-        </td>
-        <td>
-          <%= government.start_date.to_fs(:govuk_date) %>
-        </td>
-        <td>
-          <%= government.end_date.to_fs(:govuk_date) if government.end_date %>
-        </td>
-      <% end %>
-    <% end %>
+<% unless @governments.present? %>
+  <p class="govuk-body">No governments have been created.</p>
+<% end %>
```

Comment:
It might be a bit more readable to just pop this in an if else block as one will always be true.

```
if @governments.blank?
  <p class="govuk-body">No governments have been created.</p>
<% else %> 
 render_table_component
<% end %>
```

Reply:
+1 to this - if no governments exists the table would still show if we don't have `if..else`